src/cpp/session/modules/SessionBuild.R

```diff
@@ -83,4 +83,7 @@ options(buildtools.with = .rs.withBuildTools)
    sapply(failures, function(e) e$name)
 })
 
+.rs.addFunction("findShinyTestsDir", function(appDir) {
+   shinytest:::findTestsDir(appDir = appDir, mustExist = FALSE, quiet = TRUE)
```

Comment:
I don't see this function available in the current CRAN release of `shinytest`. Will this require us to bump the dependency version? Are we waiting for a CRAN release?

Reply:
Ugh, I didn't realize it was new. Nice catch. I've just pushed an update that only uses it if it exists.